packages/pds/tests/_util.ts

```diff
@@ -33,120 +32,84 @@ export type TestServerOpts = {
   } else {
     plcDb = PlcDatabase.mock()
   }
-
   const plcServer = PlcServer.create({ db: plcDb })
   const plcListener = await plcServer.start()
   const plcPort = (plcListener.address() as AddressInfo).port
   const plcUrl = `http://localhost:${plcPort}`
 
-  const recoveryKey = (await crypto.Secp256k1Keypair.create()).did()
-
-  const plcClient = new plc.Client(plcUrl)
-  const serverDid = await plcClient.createDid({
-    signingKey: repoSigningKey.did(),
-    rotationKeys: [recoveryKey, plcRotationKey.did()],
-    handle: 'localhost',
-    pds: 'https://pds.public.url',
-    signer: plcRotationKey,
+  const repoSigningKey = await crypto.Secp256k1Keypair.create({
+    exportable: true,
+  })
+  const repoSigningPriv = ui8.toString(await repoSigningKey.export(), 'hex')
+  const plcRotationKey = await crypto.Secp256k1Keypair.create({
+    exportable: true,
   })
+  const plcRotationPriv = ui8.toString(await plcRotationKey.export(), 'hex')
+  const recoveryKey = (await crypto.Secp256k1Keypair.create()).did()
 
   const blobstoreLoc = path.join(os.tmpdir(), randomStr(5, 'base32'))
 
-  const cfg = new ServerConfig({
-    debugMode: true,
-    version: '0.0.0',
-    scheme: 'http',
-    hostname: 'localhost',
-    serverDid,
-    recoveryKey,
+  const port = await getPort()
+
+  const env: ServerEnvironment = {
+    port,
+    dbPostgresUrl: dbPostgresUrl,
+    dbSqliteLocation: dbPostgresUrl ? undefined : ':memory:',
+    blobstoreDiskLocation: blobstoreLoc,
+    recoveryDidKey: recoveryKey,
+    didPlcUrl: plcUrl,
+    serviceHandleDomains: ['.test'],
+    sequencerLeaderLockId: uniqueLockId(),
+    repoSigningKeyK256PrivateKeyHex: repoSigningPriv,
+    plcRotationKeyK256PrivateKeyHex: plcRotationPriv,
     adminPassword: ADMIN_PASSWORD,
     moderatorPassword: MODERATOR_PASSWORD,
-    triagePassword: TRIAGE_PASSWORD,
-    inviteRequired: false,
-    userInviteInterval: null,
-    userInviteEpoch: Date.now(),
-    didPlcUrl: plcUrl,
-    didCacheMaxTTL: DAY,
-    didCacheStaleTTL: HOUR,
     jwtSecret: 'jwt-secret',
-    availableUserDomains: ['.test'],
-    rateLimitsEnabled: false,
-    appUrlPasswordReset: 'app://forgot-password',
-    emailNoReplyAddress: 'noreply@blueskyweb.xyz',
-    publicUrl: 'https://pds.public.url',
-    dbPostgresUrl: process.env.DB_POSTGRES_URL,
-    blobstoreLocation: `${blobstoreLoc}/blobs`,
-    blobstoreTmp: `${blobstoreLoc}/tmp`,
-    maxSubscriptionBuffer: 200,
-    repoBackfillLimitMs: HOUR,
-    sequencerLeaderLockId: uniqueLockId(),
-    bskyAppViewEndpoint: 'http://fake_address.invalid',
-    bskyAppViewDid: 'did:example:fake',
-    dbTxLockNonce: await randomStr(32, 'base32'),
+    inviteRequired: false,
+    inviteEpoch: Date.now(),
+    triagePassword: TRIAGE_PASSWORD,
     ...params,
-  })
+  }
 
-  const db =
-    cfg.dbPostgresUrl !== undefined
-      ? Database.postgres({
-          url: cfg.dbPostgresUrl,
-          schema: cfg.dbPostgresSchema,
-          txLockNonce: cfg.dbTxLockNonce,
```

Comment:
I can't think of anything that allows us to drop this—is it possible we need to bring it back?  Or maybe I missed the change!

Reply:
It got moved to `AppContext` 👍